config/config.msft.yaml

```diff
@@ -30,6 +29,12 @@ defaults:
     etcd:
       kvName: arohcp-etcd-{{ .ctx.regionShort }}
       kvSoftDelete: true
+    istio:
+      url: "https://istio.io/downloadIstio"
+      istioctlVersion: "1.23.1"
+      tag: "prod-stable"
+      newVersion: ""
```

Comment:
IIRC @geoberle had suggested `targetVersion` and a versions array - are you making that change here?